### ISSUE TITLE
WebForms Examples Explorer issues in Ext.NET 4.2.1

### ISSUE THREAD

**Issue by fabriciomurta** [extnet/examples5.ext.net#15]
## Summary

Here we will enumerate all issues on Ext.NET 4.2.1 regarding the WebForms version of the Examples Explorer. MVC Examples explorer did not raise any error on this release.

This version includes ExtJS 6.2.1. The previous Examples Explorer Webforms release issue was #9 .

## OPEN

Items here are identified as defects but not yet analysed nor fixed.

| Item | State | Description |
| --- | --- | --- |
| | | |

## PENDING

Items here are pending further analysis, another fix or something else that prevents them to be actually fixed.

| Item | State | Description |
| --- | --- | --- |
|  |  |  |

## WON'T FIX or NOT BUG

| Item | State | Description |
| --- | --- | --- |
| 354. | HID | http://localhost:51299/Examples/Keys/Panel_Keys/Window_Size/ |
| | | JavaScript error upon loading page. |
| | | Example hid until issue extnet/Ext.NET#1466 is fixed. |
| | | |

## FIXED

As items are fixed, they are moved to this section.
Items with the **FIXED**\* tag (with an asterisk) means they were fixed merely by the ExtJS upgrade or already fixed as the examples explorer was tested.

| Item | State | Description |
| --- | --- | --- |
| 22. | FIXED | http://localhost:51299/Examples/Chart/Bar/3D/ |
| | | JavaScript error upon loading page. |
| | | |
| 97. | FIXED | http://localhost:51299/Examples/Data_Binding/Basic/Chaining_Stores/ |
| | | JavaScript error upon loading page. |
| | | |
| 353. | FIXED | http://localhost:51299/Examples/Keys/Panel_Keys/Grid_Rows_Delete/ |
| | | JavaScript error upon loading page. |
| | | Depends on issue #1466. Adapted with an alternative method to work. |
| | | |
| 480. | FIXED | http://localhost:51299/Examples/SignalR/Basic/StockTicker/ |
| | | JavaScript error upon loading page. |
| | | |

**Comment by fabriciomurta** [comment]
The stock ticker example (item 408) was broken because a setting in the project file was preventing owin-related class Startup.cs from being compiled by any VS not matching versions 11, 12, or 14.
With this fix, the startup disabling call on Web.config may be removed.
